chaoswm/actions.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+from logzero import logger
```

Comment:
This import is a third-party one, like chaoslib, so should appear right after it. You may want to use [isort](https://pypi.org/project/isort/) if you want